src/ui/articleview.cc

```diff
@@ -2199,11 +2199,17 @@ void ArticleView::highlightFTSResults()
     return;
   }
 
+  QString accuracy = "exactly";
+
+  if ( regString.contains( QRegularExpression( "[\\x4e00-\\x9fa5]+" ) ) ) {
```

Comment:
```suggestion
  if ( regString.contains( QRegularExpression( R"(\p{Han})" ) ) ) {
```

The `+` sign doesn't appear to serve any purpose, because `contains` can return true with just one 1 CJK char.